docs/user/guides/build-troubleshooting.rst

```diff
@@ -1,8 +1,11 @@
-Build troubleshooting
-=====================
+Troubleshooting build errors
+============================
 
-This section is very specific and may change over time.
-Please help us keep it updated and contribute your own build errors and resolutions.
+
+.. tip::
+
+   Please help us keep this section updated and contribute your own build errors and resolutions.
+   Send in helpful ideas 💡 to support@readthedocs.org.
```

Comment:
Wouldn't it be better to open an issue/send a pull request instead?

Reply:
Definitely as a supplement :+1: Added that.

Since "Edit on GitHub" is already a present feature that technical users might know, I think that noting the support email for ideas and suggestions is also fine... to keep the bar low for adding stuff to a page like this.